msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,238 @@
+
+```powershell
+connect-MgGraph -Scopes @('TeamworkAppSettings.ReadWrite.All', 'Policy.ReadWrite.Authorization', 'AppCatalog.Read.All','InformationProtectionPolicy.Read', 'ServicePrincipalEndpoint.ReadWrite.All', 'Policy.ReadWrite.PermissionGrant', 'Policy.ReadWrite.ApplicationConfiguration', 'Application.ReadWrite.All')
+```
+
+## Enable the preapproval of RSC permissions
+
+To enable preapproval for RSC permissions, change your RSC settings with the `Set-MgBetaChatRscConfiguration` and `Set-MgBetaTeamRscConfiguration` cmdlets to allow preapproved apps. The following table list the available configurations for RSC settings:
+
+|Configuration| Description|
+|---|---|
+|`ApprovedForAllApps`|You can allow users in your tenant to access any RSC enabled app. After you enable this configuration, there's no need for preapproval of RSC permissions, as all RSC enabled apps are automatically approved for use.|
+|`ApprovedForPreApprovedAppsOnly`|RSC enabled apps can be allowed on an individual basis. Admins can determine which apps are authorized for specific RSC permissions.|
+|`DisabledForAllApps`| No RSC permissions can be consented to by users. In this case, preapproval of RSC permissions doesn't affect on users ability to consent to RSC enabled apps.|
+
+   > [!WARNING]
+   > If you change your chat or team RSC configuration to `DisabledForAllApps`, it disables preapproval in your tenant and users run into errors when they install RSC enabled apps.
+
+<br>
+<details>
+
+<summary><b>Preapproval for chat RSC permissions</b></summary>
+
+Use the following cmdlets to check and modify the current state of your chat RSC configurations.
+
+1. `Get-MgBetaChatRscConfiguration`
+1. `Set-MgBetaChatRscConfiguration -State EnabledForPreApprovedAppsOnly`
+
+</details>
+
+<br>
+<details>
+
+<summary><b>Preapproval for team RSC permissions</b></summary>
+
+Use the following cmdlets to check and modify the current state of your team RSC configurations.
+
+1. `Get-MgBetaTeamRscConfiguration`
+1. `Set-MgBetaTeamRscConfiguration -State EnabledForPreApprovedAppsOnly`
+
+</details>
+
+## Create the preapproval of RSC permissions
+
+You can create a preapproval, which preapproves RSC permissions for certain sensitivity labels within your tenant. This allows you to control the data that RSC enabled apps can access.
+
+In this section, you learn to:
+
+* [Create a preapproval without a sensitivity label](#create-the-preapproval-without-a-sensitivity-label) for use with any sensitivity group.
+* [Create a preapproval with a sensitivity label](#create-the-preapproval-with-a-sensitivity-label) for use with specific data.
+
+To create a preapproval, ensure that you have the following information:
+
+1. Teams App ID.
+1. The RSC permissions associated with the app.
+1. Team RSC permission: The sensitivity label ID associated with the sensitivity label you want to link to the preapproval policy.
```

Comment:
>Team RSC permission: [](http://example.com/codeflow?start=3&length=20)

This is confusing. What does "Team RSC permission" here mean?

Reply:
addressed in latest commit